packages/api/src/cache.ts

```diff
@@ -1,4 +1,4 @@
-import type { OASDocument } from 'oas/@types/rmoas.types';
+import type { OASDocument } from 'oas/dist/rmoas.types';
```

Comment:
Why are we importing from dist? Ideally we wouldn't have to.

Reply:
This is the `main` and `types` entry for `oas`:

```
  "main": "dist/index.js",
  "types": "dist/index.d.ts",
```

I think we wouldn't need to load it from the dist if these types were exported from the main entrypoint.

Reply:
👍